i18n/pl.toml

```diff
@@ -7,6 +7,9 @@ other = "Doświadczenie"
 [education]
 other = "Wykształcenie"
 
+[references]
+other = "Referencje"
```

Comment:
I don't speak Polish, so this is just from Google Translate, sorry

Reply:
This is OK, thanks for taking care of the translations :ok_hand: